src/components/TimeseriesDataExport/constants.ts

```diff
@@ -0,0 +1,19 @@
+export const defaultStrings = {
+  title: 'Export chart data',
+  labelRange: 'Range',
+  labelGranularity: 'Label Granularity',
+  labelGranularityHelp: 'Example inputs: 15s, 1m, 5h, 2d',
+  formatTimestamp: 'Format timestamp?',
+  formatTimestampHelp: 'e.g. 2018-04-02 12:20:20',
+  delimiterLabel: 'Select delimiter',
+  delimiterHelp: 'The character that will separate your data fields',
+  csvDownload: 'Download as CSV',
+  csvDownloadInProgress: 'Download as CSV',
+  closeBtn: 'Close',
+  imageDownloadLabel: 'Image download',
+  imageDownloadBtn: 'Download as SVG',
+  cellLimitErr:
+    'You hit the limit of {{ cellLimit }} datapoints - some data may be omitted',
+};
+
+export type Strings = typeof defaultStrings;
```

Comment:
Could you specify more "precise" type, smthing like `TimeseriesDataExportStrings`, mb we might expose it in a future

Reply:
I've kept is as `Strings` in the local scope (as the name is much shorter) and exported it as `TimeseriesDataExportStrings`, is that okay? We will probably need to type the callback function thus the export.

Reply:
yep.

Type for callback function – it's ok like it is now.